tests/build/build.go

```diff
@@ -124,9 +130,13 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 			//Delete the quay image repo since we are setting delete-repo=false
 			_, err = build.DeleteImageRepo(imageRepoName)
 			Expect(err).NotTo(HaveOccurred(), "Failed to delete image repo with error: %+v", err)
+
+			// Delete the sample private image repo created for checking inside build.IsQuayOrgSupportsPrivateRepo()
+			_, err = build.DeleteImageRepo(constants.SamplePrivateRepoName)
```

Comment:
This is fragile, since it can fail somewhere above this line.

Reply:
Agree. I think we can also cleanup the sample private repo created inside the function `IsQuayOrgSupportsPrivateRepo()` itself as suggested by @mkosiarc in the previous comment.